docs/developer/guidelines/style.md

```diff
@@ -46,4 +46,51 @@ A `flake8` plugin for ensuring a consistent logging format. We enable:
 
 ## [Mypy][mypy-github]
 
-A type checker allowing a mix of dynamic and static typing. This is optional for now.
+A comment-based type checker allowing a mix of dynamic and static typing. This is optional for now. In order to enable `mypy` for a specific integration, open its `tox.ini` file and add the 2 lines in the correct section:
+
+```
+[testenv]
+dd_check_types = true
+dd_mypy_args = <FLAGS> datadog_checks/ tests/
+...
+```
+
+The `dd_mypy_args` can be used to add or overwrite some flags to mypy for this specific integration. Here are some useful arguments you can add:
```

Comment:
We should probably link to the mypy docs here for reference, eg:

```suggestion
The `dd_mypy_args` defines the [mypy command line options][mypy-command-line] for this specific integration. Here are some useful arguments you can add:
```

With `[mypy-command-line]: https://mypy.readthedocs.io/en/stable/command_line.html`